schemas/internal/embeddings/json-schema-embeddings.json

```diff
@@ -0,0 +1,57 @@
+{
+  "title" : "Core Entry",
+  "description": "JSON schema for storing embedding data.",
+  "type": "object",
+  "required": [
+    "rcsb_accession_info",
+    "rcsb_id",
+    "embedding"
+  ],
+  "properties": {
+    "rcsb_id": {
+      "type": "string",
+      "examples": [
+        "1KIP"
+      ],
+      "description": "A unique identifier for each object in this entry container."
+    },
+    "rcsb_embedding_info": {
```

Comment:
It is named "rcsb_accession_info" in the required section above.

Reply:
Thank you @josemduarte. This is now fixed.